fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/BrokerConfig.java

```diff
@@ -34,6 +34,9 @@ public class BrokerConfig extends ConfigBase {
     @ConfField
     public static int broker_ipc_port = 8000;
 
+    @ConfField
+    public static boolean disable_check_client_expiration_task = false;
```

Comment:
1. add some documents to explain this configuration
2. add comments here to explain the background that this configuration is added. Otherwise, nobody knows why there is a configuration.

Reply:
disable_check_client_expiration_task -> disable_broker_client_expiration_checking